### PR TITLE
Rework go1

### PR DESCRIPTION
**Pull request by Hreherch**
Kept old files just in case we needed them for later (we can delete when we're sure everything is good)

to play, open gogui

then follow:
```
game > rules > attach
```
to attach the rules of Gomoku to the game
you should now be able to play Gomoku with yourself

to start a robutt:
```
program > attach
```
then you can select the robutt to play both colours or w/e

### CODE REVIEW

File: Go0.py [hunk 1]
```diff
@@ -12,11 +12,11 @@
 from board_util import GoBoardUtil
 from simple_board import SimpleGoBoard
 
-class Go0():
+class Go1():
```
Comment (MandyMeindersma) [inline Go0.py]: Here?

File: Go0.py [hunk 2]
```diff
@@ -26,18 +26,19 @@ def __init__(self):
...
+        return GoBoardUtil.generate_random_move(board, color)
     
+
 def run():
     """
     start the gtp connection and wait for commands.
     """
     board = SimpleGoBoard(7)
-    con = GtpConnection(Go0(), board)
+    con = GtpConnection(Go1(), board)
```
Comment (MandyMeindersma) [inline Go0.py]: Here

File: Go0.py [hunk 3]
```diff
@@ -26,18 +26,19 @@ def __init__(self):
         version : float
             version number (used by the GTP interface).
         """
-        self.name = "Go0"
+        self.name = "Go1"
```
Comment (MandyMeindersma) [inline Go0.py]: Here

**Comment by MandyMeindersma** [comment]
@amalik2 you don't have your gitconfig right, how are you supposed to get those precious green dots on your profile if you dont configgggg right

**Comment by amalik2** [comment]
Too much work @MandyMeindersma

**Pull request merged**